src/inputs.ts

```diff
@@ -31,15 +33,17 @@ export class ActionConfig {
   public readonly svgoOptionsPath: string;
 
   constructor(inputs: Inputs, config: RawActionConfig = { }) {
-    this.isDryRun = ActionConfig.getDryRunValue(inputs, config);
+    const isDryRun = ActionConfig.getDryRunValue(inputs, config);
+    this.isDryRun = isDryRun;
 
     this.comment = ActionConfig.getCommentValue(inputs, config);
+    this.commit = ActionConfig.getEnableCommitting(inputs, config, isDryRun);
```

Comment:
I think this should be renamed.

```suggestion
    this.commit = ActionConfig.getEnableCommits(inputs, config, isDryRun);
```

<sup>Don't forget to rename it everywhere</sup>